proposals/context.md

```diff
@@ -126,40 +129,41 @@ An element which wishes to receive some context and participate in the Context A
 const coolThingContext = createContext('cool-thing');
 
 this.dispatchEvent(
-    new ContextEvent(
-        coolThingContext, // the context we want to retrieve
-        callback: (coolThing) => {
-            this.myCoolThing = coolThing; // do something with value
-        }
-    )
+  new ContextEvent(
+    coolThingContext, // the context we want to retrieve
+    (coolThing) => {
+      this.myCoolThing = coolThing; // do something with value
+    }
+  )
 );
 ```
 
 If a provider listening for this event can provide the requested context it will invoke the callback passed in the payload of the event. The element can then do whatever it wishes with this value.
 
-It may also be the case that a provider can retain a reference to this callback, and can then invoke the callback multiple times. In this case providers should pass a dispose function as a second argument to the callback to allow consumers to inform the provider that it should no longer update the element, and should dispose of the callback.
+It may also be the case that a provider can retain a reference to this callback, and can then invoke the callback multiple times. In this case providers should pass a unsubscribe function as a second argument to the callback to allow consumers to inform the provider that it should no longer update the element, and should dispose of the callback.
```

Comment:
```suggestion
It may also be the case that a provider can retain a reference to this callback, and can then invoke the callback multiple times. In this case providers should pass an unsubscribe function as a second argument to the callback to allow consumers to inform the provider that it should no longer update the element, and should dispose of the callback.
```